utilities/tools/Test-ModuleLocally.ps1

```diff
@@ -247,30 +251,31 @@ function Test-ModuleLocally {
                 managementGroupId = $ValidateOrDeployParameters.ManagementGroupId
                 Verbose           = $true
             }
+
             try {
                 # Validate template
                 # -----------------
                 if ($ValidationTest) {
                     # Loop through test parameter files
                     foreach ($moduleTestFile in $moduleTestFiles) {
-                        Write-Verbose ('Validating module [{0}] with test file [{1}]' -f $ModuleName, (Split-Path $moduleTestFile -Leaf)) -Verbose
+                        Write-Verbose ('Validating Module [{0}] with test file [{1}]' -f $ModuleName, (Split-Path $moduleTestFile -Leaf)) -Verbose
```

Comment:
Why do we change the casing? 😄 I see that also happened with a lot of variables that are not parameters. Search & Replace?

Reply:
Yes maybe. Updated